v2/api/network/v1api20241001/structure.txt

```diff
@@ -0,0 +1,110 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+github.com/Azure/azure-service-operator/v2/api/network/v1api20241001
+--------------------------------------------------------------------
+APIVersion: Enum (1 value)
+└── "2024-10-01"
+
+NetworkWatcher: Resource
+├── Owner: resources/v1apiv20191001.ResourceGroup
+├── Spec: Object (5 properties)
+│   ├── AzureName: string
+│   ├── Location: *string
+│   ├── OperatorSpec: *Object (2 properties)
+│   │   ├── ConfigMapExpressions: *core.DestinationExpression[]
+│   │   └── SecretExpressions: *core.DestinationExpression[]
+│   ├── Owner: *genruntime.KnownResourceReference
+│   └── Tags: map[string]string
+└── Status: Object (8 properties)
+    ├── Conditions: conditions.Condition[]
+    ├── Etag: *string
+    ├── Id: *string
+    ├── Location: *string
+    ├── Name: *string
+    ├── ProvisioningState: *Enum (4 values)
+    │   ├── "Deleting"
+    │   ├── "Failed"
+    │   ├── "Succeeded"
+    │   └── "Updating"
+    ├── Tags: map[string]string
+    └── Type: *string
+
+NetworkWatchersFlowLog: Resource
+├── Owner: NetworkWatcher
+├── Spec: Object (13 properties)
+│   ├── AzureName: string
+│   ├── Enabled: *bool
+│   ├── EnabledFilteringCriteria: *string
+│   ├── FlowAnalyticsConfiguration: *Object (1 property)
+│   │   └── NetworkWatcherFlowAnalyticsConfiguration: *Object (5 properties)
+│   │       ├── Enabled: *bool
+│   │       ├── TrafficAnalyticsInterval: *int
+│   │       ├── WorkspaceId: *string
```

Comment:
Looks like it might be an ARM ID? We have `WorkspaceResourceReference` below too -- should this Id field have been removed?

Reply:
Or is this some non-ARM ID format?

Reply:
It's documented as the GUID of the Workspace - which is already identified by `WorkspaceResourceReference` - so I suspect it's an output value only. I'll prune it from the spec.